riscv-asm.md

```diff
@@ -604,4 +604,4 @@ frflags rd        | csrrs rd, fflags, x0       | Read FP exception flags
 fsflags rd, rs    | csrrw rd, fflags, rs       | Swap FP exception flags
 fsflags rs        | csrrw x0, fflags, rs       | Write FP exception flags
 fsflagsi rd, imm  | csrrwi rd, fflags, imm     | Swap FP exception flags, immediate
-fsflagsi imm      | csrrwi x0, fflags, imm     | Write FP exception flags, immediate
\ No newline at end of file
+fsflagsi imm      | csrrwi x0, fflags, imm     | Write FP exception flags, immediate
```

Comment:
What did I change here? Strange...

Reply:
Perhaps it added a newline? I used the Github web editor.

Reply:
Yes, Github silently added the newline to the last line. Supposedly this is more "POSIX-ly correct". I'm leaving it.

Reply:
sgtm